workloads/statefulset/README.md

```diff
@@ -0,0 +1,43 @@
+# StatefulSet Workload
+
+oam-kubernetes-runtime also support K8s native resource, this section will introduce how to use K8s StatefulSet.
+
+## Update RBAC for OAM AppConfig Controller
+
+Make sure your role has below rules:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: <your-role>
+rules:
+- apiGroups:
+  - apps
+  resources:
+  - statefulsets
+  verbs:
+  - *
+```
+
+## Install Component
```

Comment:
Do we need to say `please enter folder workloads/statefulset` or `cd workloads/statefulset` before we execute `kubectl apply`?